guillotina/db/writer.py

```diff
@@ -56,7 +56,7 @@ def old_serial(self):
     def part(self):
         return getattr(self._obj, "__partition_id__", 0)
 
-    def serialize(self):
+    async def serialize(self):
```

Comment:
Does this need to change?

Reply:
I didn't add the same logic as in the deserialization function (because it's expensive to know if a python object it's big or small) but I changed the function to async just in case someone wants to use a custom logic to decide when to run_async. For example:
 
```python
async def serialize(obj):
    if isinstance(obj, BigObject):
        return await run_async(pickle.dumps, obj)
    else:
        return pickle.dumps(obj)
```

Reply:
But if we feel it's not necessary I can revert this part